scripts/extract_documents.py

```diff
@@ -0,0 +1,66 @@
+""" A script to extract raw and rendered versions of all documents in the db
+
+This script extracts documents from the db into a testdocs/ directory.
+It is intended for creating a test corpus and verifying that changes
+to kuma and/or kumascript do not cause rendering changes to the documents.
+
+In order to make it work it may be necessary to modify the KUMASCRIPT_TIMEOUT
+setting in kuma/settings/common.py to be at least 60 seconds
```

Comment:
[django-constance](https://django-constance.readthedocs.io/en/latest/) is used to set variables from the database, with the default values in the settings file. The preferred way to adjust the setting is in the Django admin or with ``./manage.py constance set KUMASCRIPT_TIMEOUT 60``.

I find environment variables to be a better way to configure Django dynamically, and it is a personal goal to get rid of Constance or drastically reduce its usage. ``KUMASCRIPT_TIMEOUT`` is one of the harder ones to transition, because a lot of tests use ``KUMASCRIPT_TIMEOUT=0`` to avoid live calls to the kumascript service during tests.